frontend/src/metabase/query_builder/components/view/ViewHeader.jsx

```diff
@@ -532,3 +543,40 @@ function ViewTitleHeaderRightSide(props) {
 }
 
 ViewTitleHeader.propTypes = viewTitleHeaderPropTypes;
+
+function getDisabledSaveTooltip(
+  question,
+  isEditable,
+  requiredTemplateTags = [],
+) {
+  if (!isEditable) {
+    return t`You don't have permissions to save this question.`;
```

Comment:
I don't want to block the PR because of this, but please note that this string doesn't exist in the translation files. It will not be translated automatically and will require all OSS translation contributors to add another translation.

Was it really that important to change?
I'd double check with the product before shipping this.

Reply:
I understand that this translation doesn't exist yet — similar with the other new or updated strings.
Was it necessary or not, I don't know, in my opinion it's better to fix anything small like this. I am checking with the product about this though, thanks.

Reply:
Actually I was wrong about it and `you don't have permission to` is grammatically correct. Thank you for noticing this!